src/index/formatFileStructure/removeEmptyFolders.ts

```diff
@@ -1,18 +1,28 @@
 import fs from "fs";
 import path from "path";
 
-export const removeEmptyFolders = (p: string) => {
-  const files = fs.readdirSync(p);
-  if (!files) {
-    fs.rmdirSync(p);
-  }
-  for (const file of files) {
-    const fullPath = path.join(p, file);
-    if (fs.lstatSync(fullPath).isDirectory()) {
-      removeEmptyFolders(fullPath);
-      if (!fs.readdirSync(fullPath).length) {
-        fs.rmdirSync(fullPath);
-      }
-    }
+/**
+ * @summary
+ * Recursively removes all empty folders.
+ *
+ * @param directory the full path of a directory.
+ *
+ * @example
+ * removeEmptyFolders('/home/user/documents/project/src')
+ */
+export function removeEmptyFolders(directory: string): void {
+  const files = fs.readdirSync(directory);
+  if (!files) return fs.rmdirSync(directory);
+
+  for (const base of files) {
```

Comment:
I think the variable name `base` is a bit confusing here. We commonly use `filePath` instead.

```suggestion
  for (const filePath of files) {
```

Reply:
I used base as that is what `path.parse()` would return on a file.

Happy to change it. Would you still like it changed?